src/aiida/manage/configuration/profile.py

```diff
@@ -19,6 +19,7 @@
 from aiida.common import exceptions
 
 from .options import parse_option
+from .settings import DAEMON_DIR, DAEMON_LOG_DIR
```

Comment:
@danielhollas I assume move the modules from inside aiida will not influence too much of import time, since I think anyway the `.setting` module will be loaded. Especially for the very inner base modules like `.setting`.
The lazy load PR you implemented I guess was mostly for 3rd part libs we used, correct?

Reply:
This is fine, `.settings` is anyway imported in `src/aiida/manage/configuration/__init__.py` so this doesn't actually change anything.